wlib/SourceView.cpp

```diff
@@ -271,7 +271,6 @@ int CSourceCtrl::HandleNotify(Scintilla::SCNotification *notification)
         }
         MatchBrace();
         UpdateStatusBar();
-        m_owner->NewSel();
```

Comment:
It turns out that this was done to make sure there was a default element chosen when opening the color dialog. But this is not the way to do it because it is called each time the sourceview window in the color dialog is scrolled. I will look to call this function in the color dialog init function.